tests/simulation/test_homogenization.py

```diff
@@ -11,28 +13,29 @@ def test_homogenization():
     # values are kind of made up but within the expected magnitude
 
     # paste data
-    parameters['paste_E'] = 30e9  # Pa
+    parameters['paste_E'] = 30e9 * ureg('Pa') # Pa
```

Comment:
I would personally remove the unit in the comment, but not required

Reply:
I agree. Done.